posttroll/subscriber.py

```diff
@@ -413,3 +425,54 @@ def handle_msg(self, msg):
         else:
             LOGGER.debug("Removing address %s", str(addr_))
             self.subscriber.remove(addr_)
+
+
+def dict_config(settings):
```

Comment:
I think this name is missleading, could it be renamed to eg `create_subscriber_from_dict_config`?

Reply:
I just went with the same naming I earlier added to `posttroll.publisher`. I'll check how much I've started to use `publisher.dict_config()` on other packages and rename both if it isn't in use yet.

Reply:
Sorry for not catching this earlier.

Reply:
Seems that the only place is the [draft PR](https://github.com/pytroll/pytroll-collectors/pull/102) in pytroll-collectors so the naming change is an easy one.

Reply:
Done.